build-system/tasks/extension-helpers.js

```diff
@@ -430,14 +432,21 @@ async function buildExtension(
   if (options.compileOnlyCss && !hasCss) {
     return;
   }
-  const path = 'extensions/' + name + '/' + version;
+  const extensionPath = 'extensions/' + name + '/' + version;
```

Comment:
This makes me so happy. Variables named `path` that aren't the default `path` library make me nervous.

Reply:
Same!